test/utils/test_zenith_angle.py

```diff
@@ -61,3 +66,118 @@ def test_zenith_angle_array():
     lon = np.array([0.0])[None, None, :]
     out = cos_zenith_angle_from_timestamp(timestamp, lon, lat)
     assert out.shape == (3, 2, 1)
+
+
+@pytest.mark.parametrize(
+    "t",
+    [
+        datetime(2020, 7, 6, 9, 0, 0),
+        datetime(2000, 1, 1, 12, 0, 0),
+        datetime(2000, 7, 1, 12, 0, 0),
+        datetime(2000, 7, 1, 12, 0, 0, tzinfo=utc),
+    ],
+)
+def test_timestamp_to_julian_centuries(t):
+    a = _datetime_to_julian_century(t)
+    b = _timestamp_to_julian_century(t.replace(tzinfo=utc).timestamp())
+    assert a == b
+
+
+def test_toa():
+    t = datetime(2000, 7, 1, 12, 0, 0, tzinfo=utc).timestamp()
+    lat, lon = 0.0, 0.0
+    ans = toa_incident_solar_radiation_accumulated(t, lat, lon)
+    assert ans >= 0
+
+
+def test_tisr_matches_cds():
```

Comment:
This was helpful for validating against IFS, but I can remove it from the PR if we are comfortable with the skill.